Classes/Private/NSAttributedString/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -259,4 +268,8 @@ class HMTLNodeToNSAttributedString: SafeConverter {
     private func isUnderlined(node: ElementNode) -> Bool {
         return node.name == HTMLTags.U.rawValue
     }
+
+    private func isBlockquote(node: ElementNode) -> Bool {
+        return [HTMLTags.Blockquote.rawValue].contains(node.name)
```

Comment:
A bit nit-picky, but for consistency: when there's a single element to compare agains I'm using just the `==` operator above.